test/v4/orderList.js

```diff
@@ -1821,3 +1865,7 @@ async function debugOrders(max) {
             );
     }
 }
+
+function log(str) {
```

Comment:
this func is not used again

Reply:
well
I am thinking anytime I start debugging I will use it.
so its easier for me it just stays there....

Think its not too ugly...